metabase/metabase
# Fix rendering markdown links

Fixes #18114: markdown dashboard cards were not displaying plain text links, only markdown ones. A regression since the webpack upgrade to v5 (#16113), which required upgrading [react-markdown](https://github.com/remarkjs/react-markdown) to v6 (#16934). React Markdown dropped built-in plain text links rendering in favor of [remark-gfm](https://github.com/remarkjs/remark-gfm) plugin, see an example issue [here](https://github.com/remarkjs/react-markdown/issues/634). Fixed by installing and enabling `remark-gfm` plugin.

### To Verify

1. Go to dashboard > add Text box and input the following, then save

```
- Visit https://www.metabase.com
- Or go to [Metabase](https://www.metabase.com)
```

2. Ensure both links are rendered as HTML `<a />`

### Demo

**Before**

![135266489-125e4d41-46c3-4c88-99f8-6971740a1d39](https://user-images.githubusercontent.com/17258145/136982011-e7c30151-eee1-4792-9a8d-5b983c651f98.png)

**After**

![135266556-2f5b8508-34d3-48a5-b68a-121cfa2d0e84](https://user-images.githubusercontent.com/17258145/136981892-6773835f-53bc-4f51-a7d4-f29629820881.png)
